core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -124,6 +124,7 @@ public int run(final String[] args, final Properties config) {
         } catch (final Throwable e) {
             exitCode = EXIT_CODE_ERROR;
             System.err.println("ERROR: " + e.getMessage());
+            e.printStackTrace(System.err);
```

Comment:
In line 126 above, maybe we can just call `e.toString` which will include the message string if it contains? otherwise LGTM